OrcanodeMonitor/Models/OrcanodeEvent.cs

```diff
@@ -89,13 +90,15 @@ public OrcanodeEvent(Orcanode node, string type, string value, DateTime timestam
         /// <summary>
         /// Foreign Key for an Orcanode.
         /// </summary>
-        public int OrcanodeId { get; set; }
+        public string OrcanodeId { get; set; }
 
         // Navigation property that uses OrcanodeId.
         public virtual Orcanode Orcanode { get; set; }
 
         public DateTime DateTimeUtc { get; set; }
 
+        public string year { get; set; }
```

Comment:
```suggestion
        public string Year { get; set; }
```
Can this be an integer or must it be a string?
And if this is a partition key, add a comment saying so.